src/cli-sdk/src/commands/query.ts

```diff
@@ -10,17 +11,23 @@ import {
   jsonOutput,
   mermaidOutput,
 } from '@vltpkg/graph'
+import { error } from '@vltpkg/error-cause'
 import { Query } from '@vltpkg/query'
 import { SecurityArchive } from '@vltpkg/security-archive'
 import { commandUsage } from '../config/usage.ts'
 import type { CommandFn, CommandUsage } from '../index.ts'
 import { startGUI } from '../start-gui.ts'
 import type { Views } from '../view.ts'
+import type { LoadedConfig } from '../config/index.js'
 
 export const usage: CommandUsage = () =>
   commandUsage({
     command: 'query',
-    usage: ['', '<query> --view=[human | json | mermaid | gui]'],
+    usage: [
+      '',
+      '<query> --view=[human | json | mermaid | gui]',
+      '<query> --expect-results=[number | string]',
```

Comment:
```suggestion
      '<query> --expect-results=<comparison string>',
```

Or something to clarify it's a specific _sort_ of string we're after here.